lib/ember-test-helpers/abstract-test-module.js

```diff
@@ -95,11 +95,15 @@ export default Klass.extend({
   },
 
   setupTestElements() {
-    if (!document.querySelector('#ember-testing')) {
+    let testEl = document.querySelector('#ember-testing');
+    if (!testEl) {
       let element = document.createElement('div');
       element.setAttribute('id', 'ember-testing');
 
       document.body.appendChild(element);
+      this.fixture = '';
+    } else {
+      this.fixture = testEl.innerHTML;
```

Comment:
Can you change this to `this.fixtureResetValue`, and initialize it in the constructor?